docs/components/dialogs.md

```diff
@@ -0,0 +1,471 @@
+
+```dart
+SimpleDialog(
+  title: Text('Set backup account'),
+  children: [
+    SimpleDialogItem(
+      icon: Icons.account_circle,
+      color: Colors.orange,
+      text: 'user01@gmail.com',
+      onPressed: () {
+        Navigator.pop(context, 'user01@gmail.com');
+      },
+    ),
+    SimpleDialogItem(
+      icon: Icons.account_circle,
+      color: Colors.green,
+      text: 'user02@gmail.com',
+      onPressed: () {
+        Navigator.pop(context, 'user02@gmail.com');
+      },
+    ),
+    SimpleDialogItem(
+      icon: Icons.add_circle,
+      color: Colors.grey,
+      text: 'Add account',
+      onPressed: () {
+        Navigator.pop(context, 'user02@gmail.com');
+      },
+    ),
+  ],
+)
+
+class SimpleDialogItem extends StatelessWidget {
+  const SimpleDialogItem({Key key, this.icon, this.color, this.text, this.onPressed})
+      : super(key: key);
+
+  final IconData icon;
+  final Color color;
+  final String text;
+  final VoidCallback onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return SimpleDialogOption(
+      onPressed: onPressed,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Icon(icon, size: 36.0, color: color),
+          Padding(
+            padding: const EdgeInsetsDirectional.only(start: 16.0),
+            child: Text(text),
+          ),
+        ],
+      ),
+    );
+  }
+}
+```
+
+### Simple dialog anatomy and key properties
+
+A simple dialog contains the following:
+
+
+![simple dialog anatomy diagram](assets/dialogs/Dialogs_SimpleDialog_anatomy.png)
+
+1. Container
+1. Title 
+1. List item
+    1. Supporting visual
+    1. Primary text
+1. Button
+1. Scrim
+
+
+
+**Container attributes**
+
+| &nbsp; | **Properties** |
+| --- | --- |
+| **Color** | `backgroundColor` |
+| **Shape** | `shape` |
+| **Elevation** | `elevation` |
+
+
+**Title attributes**
+
+| &nbsp; | **Properties** |
+| --- | --- |
+| **Text label** | `title` |
+| **Color** | `style` on `title` when using a `Text` | 
+| **Typography** | `style` on `title` when using a `Text` |
+
+
+**List item supporting visual attributes**
+
+
+| &nbsp; | **Properties** |
+| --- | --- |
+| **Options** | `children` (Use `SimpleDialogOption` and customize its `child` parameter as needed.) |
+
+
+## Confirmation dialog
+
+Confirmation dialogs give users the ability to provide final confirmation of a choice before committing to it, so they have a chance to change their minds if necessary.
+
+If the user confirms a choice, it’s carried out. Otherwise, the user can dismiss the dialog. For example, users can listen to multiple ringtones but only make a final selection upon tapping “OK.”
+
+### Confirmation dialog example
+
+`Dialog`
+- [Class description](https://api.flutter.dev/flutter/material/Dialog-class.html)
+- [Github source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/dialog.dart)
+
```

Comment:
Is there a Dartpad demo?

Reply:
No not for Confirmation dialog, since it doesn't exist in Flutter and would require more custom code to implement

Reply:
In that case, you can delete the example and anatomy sections for confirmation dialogs.